app/Api/Auth/ApiAuthorizer.php

```diff
@@ -3,29 +3,20 @@
 namespace DevCommunityDE\CodeFormatter\Api\Auth;
 
 /**
- * Class ApiAuthorizer
- *
- * @package DevCommunityDE\CodeFormatter\Api\Auth
```

Comment:
php-cs-fixer does not like @package 
We can try to add a exception if you like, but i'm not sure if this is easily feasible. I see what i can do

Reply:
No problem, we can get rid of all `@package` comments.

Reply:
I think `@package` annotation could go away, we already got namespace. But I'm not sure.

Oh dear, another person I almost linked 😆